src/shadowbox/README.md

```diff
@@ -1,203 +1,169 @@
+### Self-Hosted Installation
 
-To install and run Shadowbox on your own server, run
+1. **Run the Installation Script**
 
-```
-sudo bash -c "$(wget -qO- https://raw.githubusercontent.com/Jigsaw-Code/outline-apps/master/server_manager/install_scripts/install_server.sh)"
-```
+   ```sh
+   sudo bash -c "$(wget -qO- https://raw.githubusercontent.com/Jigsaw-Code/outline-apps/master/server_manager/install_scripts/install_server.sh)"
+   ```
 
-You can specify flags to customize the installation. For example, to use hostname `myserver.com` and the port 443 for access keys, you can run:
+1. **Customize (Optional)**
 
-```
-sudo bash -c "$(wget -qO- https://raw.githubusercontent.com/Jigsaw-Code/outline-apps/master/server_manager/install_scripts/install_server.sh)" install_server.sh --hostname=myserver.com --keys-port=443
-```
+   Add flags for hostname, port, etc. Example:
 
-Use `sudo --preserve-env` if you need to pass environment variables. Use `bash -x` if you need to debug the installation.
+   ```sh
+   sudo bash -c "$(wget -qO- https://raw.githubusercontent.com/Jigsaw-Code/outline-apps/master/server_manager/install_scripts/install_server.sh)" install_server.sh \
+     --hostname=myserver.com \
+     --keys-port=443
+   ```
 
-## Running from source code
+   - Use `sudo --preserve-env` for environment variables.
+   - Use `bash -x` for debugging.
 
-### Prerequisites
+### Running from Source Code
 
-Shadowbox supports running on linux and macOS hosts.
+**Prerequisites**
 
-Besides [Node](https://nodejs.org/en/download/) you will also need:
+- [Docker](https://docs.docker.com/engine/install/)
+- [Node](https://nodejs.org/en/download/) LTS (`lts/hydrogen`, version `18.16.0`)
+- [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) (version `9.5.1`)
 
-1. [Docker Engine](https://docs.docker.com/engine/installation/)
+> [!TIP]
+> If you use `nvm`, switch to the correct Node version with `nvm use`.
 
-### Running Shadowbox as a Node.js app
+1. **Build and Run:**
 
-Build and run the server as a Node.js app:
+   Shadowbox supports running on linux and macOS hosts.
 
-```
-npm run action shadowbox/server/start
-```
+   - **Node.js App**
 
-The output will be at `build/shadowbox/app`.
+     ```sh
+     npm run action shadowbox/server/start
+     ```
 
-### Running Shadowbox as a Docker container
+   - **Docker Container**
 
-### With docker command
+     ```sh
+     npm run action shadowbox/docker/start
+     ```
 
-Build the image and run server:
+     > [!TIP]
+     > Some useful commands when working with Docker images and containers:
+     >
+     > - **Debug Image:**
+     >
+     >   ```sh
+     >   docker run --rm -it --entrypoint=sh localhost/outline/shadowbox
+     >   ```
+     >
+     > - **Debug Running Container:**
+     >
+     >   ```sh
+     >   docker exec -it shadowbox sh
+     >   ```
+     >
+     > - **Cleanup Dangling Images:**
+     >
+     >   ```sh
+     >   docker rmi $(docker images -f dangling=true -q)
```

Comment:
I think `docker system prune` may be a better option.

Reply:
This is a pretty brute force command without filtering to a label or name, so I would hesitate to list it here. Many developers will be have other containers, volumes, caches, etc. for other Docker projects that can be costly to prune. We should consider adding labels to all created Docker artefacts and then we can more easily provide example commands to clean up only Outline artefacts.